src/OpenTelemetry.Exporter.Prometheus.AspNetCore/README.md

```diff
@@ -47,36 +47,23 @@ dotnet add package --prerelease OpenTelemetry.Exporter.Prometheus.AspNetCore
 ### Step 3: Configure Prometheus Scraping Endpoint
 
 * Register Prometheus scraping middleware using the
-  `UseOpenTelemetryPrometheusScrapingEndpoint` extension:
+  `UseOpenTelemetryPrometheusScrapingEndpoint` extension method
+  on `IApplicationBuilder` :
 
     ```csharp
-    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
-    {
-        app.UseOpenTelemetryPrometheusScrapingEndpoint();
-        app.UseRouting();
-        app.UseEndpoints(endpoints =>
-        {
-            endpoints.MapControllers();
-        });
-    }
+    var builder = WebApplication.CreateBuilder(args);
```

Comment:
happy to see alternative suggestions. I removed `Configure` as the new asp.net core templates use a simpler program.cs. and we dont want people new asp.net (via 6.0) to scramble looking for Configure method.